fips/usr/lib/systemd/system/snap-initramfs-mounts.service.d/fips.conf

```diff
@@ -0,0 +1,10 @@
+[Unit]
+After=systemd-modules-load.service
+
+[Service]
+# Ensure tcrypt was loaded
+ExecStartPre=modprobe tcrypt
```

Comment:
But you won't need this anymore now if tcrypt is in fips.conf, right?

Reply:
I do to enforce errors. modules-load.d can contain unkown modules or those that fail to load.

Whereas under FIPS compliance prior to using cryptographic functionality selftest must complete. This modprobe ensure that this is enfoced, and if tcrypt fails to load (or any selftests failed) snap-initramfs-mounts service aborts prior to mounting anything and potentially using cryptographic functionality via dm-crypt.